ml-proto/spec/eval.ml

```diff
@@ -88,6 +88,9 @@ let numerics_error at = function
       error at "runtime: invalid conversion to integer"
   | exn -> raise exn
 
+let some_memory c =
+  match c.module_.memory with Some m -> m | _ -> assert false
```

Comment:
Maybe let's call it `Crash`. Soundness then is the property that a crash
cannot happen for a checked program. :)

Not sure why you'd want to drop the error info. There aren't many places
that raise this, so I would err on the side of being more informative in
those cases.